ConcurrencyApp/UI/FetchPostsViewController.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  ViewController.swift
+//  ConcurrencyApp
+//
+//  Created by Renato F. dos Santos Jr on 27/06/25.
+//
+
+import UIKit
+import SwiftUI
+
+final class FetchPostsViewController: UIViewController {
+    private let contentView: FetchPostsView
+    private let service: FetchPostsService
+
+    init(contentView: FetchPostsView = FetchPostsView(),
```

Comment:
Great! How about we apply the full indentation of breaking lines? For example, move the first parameter to a new line, and the same goes for the second one.
```suggestion
    init(
        contentView: FetchPostsView = FetchPostsView(),
```
You could use `CMD + M` to full identity it 😄

Reply:
Got it, I've missed this point. I'll further add SwiftLint to the projecct. 

`CMD + M` or `CMD + A` for full selection then `control + I`?